### PR TITLE
Updates for Multi-Cluster docs

### PR DESCRIPTION
**Pull request by jsturtevant**
This PR includes:

- Clean up older variables that are not used in `single-key-vault` [environment](https://github.com/microsoft/bedrock/tree/master/cluster/environments/azure-single-keyvault)
- adds references to required parts of the tutorials that are pre-reqs to create two clusters
- aligns some vnet variables to make it easy to follow
- removes and rewords versions of the template.  Bedrock does not have a release or tag for 1.0 so ended up with errors when following tutorial

cc: @NathanielRose @jred55

### CODE REVIEW

File: docs/firstWorkload/README.md [hunk 1]
```diff
@@ -191,6 +191,8 @@ Our deployment specification includes references to values for the [Azure Servic
 
 For this walkthrough, we will use one Service Principal to deploy with Terraform and for the AKS cluster itself.
 
+> In practice you will have seperate Service Principals for each with scopes set appropriatly to the correct resources. See more on [Service Principals](https://github.com/microsoft/bedrock/tree/master/cluster/azure#create-an-azure-service-principal)
```
[the remark this answers is not in the document]
Comment (jsturtevant) [reply docs/firstWorkload/README.md]: Yes, that is correct and updated it to be a bit clearer I hope

File: docs/firstWorkload/README.md [hunk 2]
```diff
@@ -191,6 +191,8 @@ Our deployment specification includes references to values for the [Azure Servic
 
 For this walkthrough, we will use one Service Principal to deploy with Terraform and for the AKS cluster itself.
 
+> In practice you will have separate Service Principals: one for deploying with terraform and one for each AKS cluster deployed.  The AKS cluster Service principle is used by cluster provider to create resources like load balancers and should only [have enough permissions to interact with the resources need by the cluster](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal#delegate-access-to-other-azure-resources).  The terraform Service Principal will require the ability to create and modify any resources in the terraform modules deployed. See more on [Service Principals](https://github.com/microsoft/bedrock/tree/master/cluster/azure#create-an-azure-service-principal)
```
Comment (jsturtevant) [inline docs/firstWorkload/README.md:194]: @dennisseah does this look better?

File: docs/single-cluster.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ az storage account create \
     --resource-group my-global-rg \
     --location eastus \
     --sku Standard_LRS \
-    --encryption blob
+    ----encryption-services blob
```
Comment (andrebriggs) [inline docs/single-cluster.md:33]: Too many `--`

**Comment by andrebriggs** [comment]
@jmspring @jsturtevant I went ahead and resolved the issues

**Pull request merged**